azerothcore/azerothcore-wotlk
# refactoring(SAI): add event_param5 and target_param4

###### ISSUES ADDRESSED:

Closes https://github.com/azerothcore/azerothcore-wotlk/issues/1197

##### TESTS PERFORMED:

Compiles, runs

##### HOW TO TEST THE CHANGES:

- **Carefully** review this Pull Request code changes
- Make sure the SmartAI system is not broken. For example, take some SmartAI script that you know well and make sure it still works as expected.


##### Target branch(es):

Master
